src/locales/en/fields.yml

```diff
@@ -49,6 +49,16 @@ address-confirm:
   previous: Previous address
   change: Change
 
+addressBreak:
+  legend: "What would you like to do?"
```

Comment:
header classes should be - `govuk-fieldset__legend--s`. However I cant seem to override the radio button legends default class. 🤔

Reply:
Is this a bug in the `hmpo-radio-buttons`?

Reply:
@merlinc Potentially, I haven't raised just yet in case I was doing something wrong, perhaps if we get 5 minutes so I can run through what I done and work from there?